code/game/machinery/computer/communications.dm

```diff
@@ -556,6 +556,10 @@
 	if (isnull(shuttle_template.who_can_purchase))
 		return FALSE
 
+	if (!(obj_flags & EMAGGED))
+		if (shuttle_template.restricted)
+			return FALSE
```

Comment:
not that this really matters but
```suggestion
	if (!(obj_flags & EMAGGED) && shuttle_template.restricted)
			return FALSE
```

Reply:
> not that this really matters but

Adding it, along with the correct indentation!